### PR TITLE
Upgrade to Spotlight v2.13.0

### LINKED ISSUE #936: Upgrade to Spotlight 2.13
Do this to get the deprecation warnings we'll need to fix for Spotlight 3.

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #936

### CODE REVIEW

File: app/values/manifest_metadata.rb [hunk 1]
```diff
@@ -128,6 +128,13 @@ def manifest_fields
     super
   end
 
+  # Override label method so it returns multiple values
```
Comment (hackartisan) [inline app/values/manifest_metadata.rb:131]: Can we add a canonical link to the overridden implementation?

**Review by hackartisan — CHANGES_REQUESTED** [review]
🎉 

I think we just want a little bit more on the override comment

**Review by hackartisan — APPROVED** [review]

**Comment by tpendragon** [comment]
Is json_ld_value really not used in `iiif_manifest`?

**Comment by eliotjordan** [comment]
@tpendragon That's correct. It's only used in the metadata class (Spotlight::Resources::IiifManifest::Metadata or, in our case, .ManifestMetadata).

**Pull request merged**